assets/src/editor/events/tickets/editor-ticket/grid-view/editor-ticket-grid-item.js

```diff
@@ -125,6 +139,12 @@ class EditorTicketGridItem extends Component {
 					eventDateTicketMap={ eventDateTicketMap }
 					doRefresh={ doRefresh }
 				/>
+				<EntityLock
+					entity={ ticket }
+					isLocked={ !! ticket.locked }
```

Comment:
haven't yet decided fully on a name for the property.

initially I was thinking of just having a boolean, in which case `isLocked` would be perfect, but then I started thinking that there may be different levels of "lockiness". For example:

- an "edit lock" that merely prevents bulk changes from being applied to entities that have been customized
- a "price lock" that prevents the price of a ticket from being changed because it has already been sold and we want to prevent accounting discrepancies

the former may allow the admin to manually toggle the lock on and off as it is primarily intended to block bulk changes from being applied to an entity, whereas the latter would allow edits to the entity's properties as long as they don't affect the price. As well, an admin would NOT be able to toggle a price lock off and then edit the price. It would be a permanent locking off the price.

Anyways, a lock property that can have multiple levels / varying values other than a simple on/off state would not be described well by `isLocked`.

Ultimately something as simple as just `lock` may work best.

Reply:
It would be good if you do the isLocked/locked or whatever api you want to do in its own pull.  It helps focus the discussion on just one api and that way tests can be written for it as well.   I understand you may want to test any api out with the REM add-on to make sure it works for your needs, so it'd be understandable that you'd test this (along with other things) in a separate branch.  However, once you solidify on the api, it's _super_ helpful if you then extract that work into its own pull for review.